### ISSUE TITLE
[QUESTION] How to tag stratospheric ozone with UCX enabled?

### ISSUE THREAD

**Issue by xpye98** [geoschem/geos-chem#1127]
### Ask a question about GEOS-Chem:
In tagged ozone simulation, stratospheric ozone is tagged using the Linoz mechanism. However, for new GC full-chemistry simulation, strat O3 is calculated online with UCX.

So my question is, how to tag strat O3 now? Tagged ozone simulation may give a different result from that from full-chemistry (Linoz vs. UCX). Or is there a way to just turn off UCX in full-chemistry to do a sensitivity test to know the strat ozone?

**Comment by yantosca** [comment]
Thanks for writing @xpe98.  What you can do is this:

  1. Archive the PO3 and LO3 rates from your full-chem simulation with UCX.
  2. Define stratospheric ozone as PO\3 above the local tropopause.

The tagged Ox simulation has gotten a bit out of date with the rest of the model.  Thanks for your patience as we sort out some of these issues.

**Comment by xpye98** [comment]
@yantosca Thanks for your reply!

Yes, I think that's the way to solve it. 

In `GeosCore/tagged_o3_mod.F90`, I did some modifications as below, hope you can check if it is correct.

(1) Line 186
```diff
- ! Skip stratospheric boxes
- IF ( .not. ITS_IN_TROP ) RETURN
+ ! DO NOT skip stratospheric boxes
+ ! IF ( .not. ITS_IN_TROP ) RETURN
```

(2) Around line 251, I add start O3 as what we do for other regions. Also, I think here we need to turn off linoz in `input.geos` file, otherwise the tagged simulation will calculate strat O3 using linoz again.
```diff
+ !-----------------------
+ ! P(O3) in stratosphere
+ ! One must turn off linoz in input.geos file
+ !-----------------------
+ IF ( .not. ITS_IN_TROP ) THEN
+    PP(I,J,L,2) = PPROD
+ ENDIF
```

(3) Around line 527. Here, I'm not sure if I did it correctly. I tried still using zero loss, and the result of strat [O3] can be very large. I'm confused why here the LL for strat boxes was zero before, is it because the PP for strat was zero?
```diff
- IF ( State_Met%InTroposphere(I,J,L) ) THEN
-    LL = Spc(I,J,L,N) * L24H(I,J,L) * BOXVL * DT
- ELSE
-    LL = 0.0e+0_fp
- ENDIF

+ LL = Spc(I,J,L,N) * L24H(I,J,L) * BOXVL * DT
```

**Comment by yantosca** [comment]
Note that 13.4.0 will include an updated tagged O3 simulation that fixes several of these issues.  See #1150.

**Comment by yantosca** [comment]
I will close out this issue as an updated tagged O3 simulation has already been merged into the 13.4.0 development stream.  Version 13.4.0 will be released as soon as we conclude the benchmark approval process (we are sorting out some issues in the full-chemistry simulation).
